source/simplex/polynomials.cc

```diff
@@ -442,6 +556,10 @@ namespace Simplex
     if (grads.size() == this->n())
       for (unsigned int i = 0; i < this->n(); i++)
         grads[i] = compute_grad(i, unit_point);
+
+    if (grad_grads.size() == this->n())
+      for (unsigned int i = 0; i < this->n(); i++)
+        grad_grads[i] = compute_grad_grad(i, unit_point);
```

Comment:
In particular, the error is higher now because previously all the hessians were just set to zero.